### DISCUSSION TITLE
Add a way to share text and media between files

### DISCUSSION THREAD

**Discussion by ImUrX** [flatpak/xdg-desktop-portal#1431] [New Portals]
There is a feature used in most phones which is the share button which has been implemented also on Windows and macOS.

It would be nice to have a way to do this in Linux also, but it seems it would need quite a bit of work as apps should have to say that they support this kind of input probably via their `.desktop` file or something like that.

**Comment by rusty-snake** [comment]
You mean like #12?

**Comment by ImUrX** [comment]
oh, thank you! I exclusively looked for it in discussion section so i couldnt find it
